config/webpack.common.js

```diff
@@ -159,6 +160,19 @@ module.exports = function (options) {
           use: ['json-loader']
         },
 
+        /* HTML Linter
+         * Checks all files against .htmlhintrc
+        */
+        {
+          enforce: 'pre',
+          test: /\.html$/,
+          loader: 'htmlhint-loader',
+          exclude: [/node_modules/,/src\/a-runtime-console/,/src\/app\/space\/wizard/],
```

Comment:
@mindreeper2420 since you use `StyleLintPlugin` with it ignore file do we need to specify the excluded directories here too?

Reply:
@corinnekrych this is specifically for the HTML files, while the stylelint ignore file is only handling the `.less` files. Removing these ignores will throw errors in the build, as well as force each build to check all of our node modules.

Reply:
+1 thanks for the explanation!